resources.html

```diff
@@ -108,12 +108,28 @@ <h4> Have a good resource for your fellow researchers to use? Feel free to add i
                 <br>
                 <br>
 
-                <!-- Dummy iframe to prevent page from redirecting to our API on submission -->
-                <iframe name="hiddenFrame" width="0" height="0" border="0" style="display: none;"></iframe>
-                <form action="https://submarine15fr43wae541f.herokuapp.com/resources" method="POST" target="hiddenFrame">
+                <script>
+                    function submitResource() {
+
+                        const url = document.querySelector('input[name="url"]').value;
+
+                        $.post({
+                            url: 'https://submarine15fr43wae541f.herokuapp.com/resources',
+                            xhrFields: {
+                                withCredentials: true,
+                            },
+                            data: {
+                                url: url,
+                            },
+                        }).done(() => {
+                            console.log('successfully submitted the url', url);
+                        });
+                    }
+                </script>
+                <div>
                     <input type="url" name="url" placeholder="Enter a URL here" required />
-                    <button type="submit">Submit Resource</button>
-                </form>
+                    <button type="button" onclick="submitResource()">Submit Resource</button>
+                </div>

```

Comment:
Like we discussed you'll eventually want to use a `<form>` again so people can hit <kbd>ENTER</kbd> and be able to submit the form, etc